### PR TITLE
Add deactivate device action

### LINKED ISSUE #1403: Add information about device deactivation
This section talks about inactive device:
https://www.balena.io/docs/learn/manage/billing/#inactive-devices

It redirects to device actions page for more info:
https://www.balena.io/docs/learn/manage/actions/#device-specific-actions

However, the latter has no info about device deactivation.

### PR DESCRIPTION
**Pull request by garethtdavies**
Closes #1403

Change-type: patch
Signed-off-by: Gareth Davies <gareth@balena.io>

### CODE REVIEW

File: pages/learn/manage/actions.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ This action allows you to remotely update the host OS running on your device. Fo
 
 Turning on local mode is useful when you are prototyping your application, as it allows you to push changes to your device over the local network without relying on the {{ $names.company.lower }} build pipeline. You can find more information in our [development guide][local-mode].
 
+### Deactivate Device
+
+This action will [deactivate the device][inactive-devices] and charge a one-time deactivation fee. Note that the device must be offline to be able to deactivate the device.
```
Comment (xginn8) [inline pages/learn/manage/actions.md:93]: outside the scope here but i'm kind of curious the heuristic you use for inline notes vs special blocked notes

Comment (garethtdavies) [reply pages/learn/manage/actions.md:93]: That's a good observation and hadn't given it much thought and using in the colloquial sense. I'll actually remove the "Note that" as "The device must be offline to be able to deactivate the device." is correct.

**Comment by garethtdavies** [comment]
Updated this to add full conditions for deactivating the device.

**Comment by garethtdavies** [comment]
@balena-ci rebase

**Pull request merged**